pkg/3scale/amp/component/apicast_options.go

```diff
@@ -26,13 +26,38 @@ func (c CustomPolicy) AnnotationKey() string {
 	// prefix/name: value
 	// The name segment is required and must be 63 characters or less
 	// Currently: len(CustomPoliciesAnnotationKeyPrefix) + 32 (from the hash) = 54
```

Comment:
change the comment as well

Reply:
Unfortunately with the current code for custom environment there's no variable that reflects that part unless we split it as I've done with the opentracing annotation. That comment was originally wrong already.
If you want I can split it as I've done with the opentracing config to make it accurate

Reply:
Ok, I will fix that in the other PR about custom envs. 

Yes, I am aware that originally the comment was wrong.

Reply:
:+1: updated